generate/types.go

```diff
@@ -22,3 +22,13 @@ type InputItem struct {
 	// used for checking the provided IP for each input type.
 	CIDR map[string][]string `yaml:"cidr"`
 }
+
+// CidrDataOutput is the output of the generate-index tool
+type CidrDataOutput struct {
```

Comment:
This struct looks redundant. I think we can add `json` tags to the `generate.Input` struct and use it directly.

Reply:
Using `generate.Input` struct outputs json in a  different format i.e. 
  ```
  "cdn": {
        "cidr": {
            "cloudfront": [
                "120.52.22.96/27",
                "205.251.249.0/24",
		 ]
	      }
   	 }
}
```
Reused the `cdncheck.InputCompiled` struct with `json `tags to avoid redundancy.